### PR TITLE
LRQA-67918 - Create test stubs for Workflow Metrics - All Items (Reassign Tasks)

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by rodrigocunhaa** [comment]
ci:forward

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsListReassignTasks.testcase [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Verify that the user can reassign several tasks to a different users and All Items page is updated accordingly"
```
Comment (rodrigocunhaa) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsListReassignTasks.testcase:32]: @evanilson-santana ,

Could you please also add the Issue ID (column I on the Test Map) on the description?

Like this:
@description = "LPS-102664 - Verify that the user can reassign several tasks to a different user and All Items page is updated accordingly"

Please, do it for all the other descriptions too.

Thanks!

**Review by rodrigocunhaa — CHANGES_REQUESTED** [review]
Hi @evanilson-santana ,

Please, make the requested changes.

Thanks!

**Comment by evanilson-santana** [comment]
Fixed.

**Comment by rodrigocunhaa** [comment]
ci:forward

**Pull request closed without merging**